src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Styles/GridSplitter.xaml

```diff
@@ -0,0 +1,69 @@
+<!--
+    This Source Code Form is subject to the terms of the MIT License.
+    If a copy of the MIT was not distributed with this file, You can obtain one at https://opensource.org/licenses/MIT.
+    Copyright (C) Leszek Pomianowski and WPF UI Contributors.
+    All Rights Reserved.
+    
+    Based on Microsoft XAML for Win UI
+    Copyright (c) Microsoft Corporation. All Rights Reserved.
+-->
+
+<ResourceDictionary 
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+
+    <system:Double x:Key="GridsplitterThumbHeight">24</system:Double>
+    <system:Double x:Key="GridsplitterThumbWidth">4</system:Double>
+    <system:Double x:Key="GridsplitterThumbRadius">2</system:Double>
+    <Thickness x:Key="GridsplitterPadding">4</Thickness>
+
+    <Style x:Key="DefaultGridSplitterStyle" TargetType="{x:Type GridSplitter}">
+        <Setter Property="IsTabStop" Value="True" />
+        <Setter Property="HorizontalAlignment" Value="Stretch" />
+        <Setter Property="VerticalAlignment" Value="Stretch" />
+        <Setter Property="MinHeight" Value="8" />
+        <Setter Property="MinWidth" Value="8" />
```

Comment:
I think three should we have GridSplitterMinWidth and GridSplitterMinHeight resources for MinWidth and MinHeight properties ?